app/views/dashboard/_activity.njk

```diff
@@ -0,0 +1,77 @@
+<div class="grid-row">
+  <form class="column-half" action="/" method="get">
+    <fieldset class="form-group">
+      <label for="activity-period" class="visually-hidden">Select a date range</label>
+      <select id="activity-period" name="period" class="form-control">
+        <option {% if period == 'today' %}selected{% endif %} value="today">Today</option>
+        <option {% if period == 'yesterday' %}selected{% endif %} value="yesterday">Yesterday</option>
+        <option {% if period == 'previous-seven-days' %}selected{% endif %} value="previous-seven-days">Previous 7 days</option>
+        <option {% if period == 'previous-thirty-days' %}selected{% endif %} value="previous-thirty-days">Previous 30 days</option>
+      </select>
+      <noscript>
+        <button class="button nojs-update-button" type="submit">Update</button>
+      </noscript>
+    </fieldset>
+  </form>
+</div>
+
+{% set transactions_period_string %}
```

Comment:
Should we put that variable in the controller? Or it is actually "local view" variable and belongs more to the view logic?